src/Backend/Modules/Faq/Domain/Category/Category.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace Backend\Modules\Faq\Domain\Category;
+
+use Common\Doctrine\Entity\Meta;
+use Common\Locale;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Exception;
+
+/**
+ * @ORM\Entity(repositoryClass="Backend\Modules\Faq\Domain\Category\CategoryRepository")
+ * @ORM\Table(name="faq_categories")
+ */
+class Category
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var Locale
+     *
+     * @ORM\Column(type="locale")
+     */
+    private $locale;
+
+    /**
+     * @var Meta
+     *
+     * @ORM\OneToOne(targetEntity="Common\Doctrine\Entity\Meta", cascade={"persist","remove"}, orphanRemoval=true)
+     */
+    private $meta;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     */
+    private $extraId;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $title;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     */
+    private $sequence;
+
+    /**
+     * @var Collection
+     *
+     * @ORM\OneToMany(
+     *     targetEntity="Backend\Modules\Faq\Domain\Question\Question",
+     *     mappedBy="category",
+     *     cascade={"remove"}
+     * )
+     */
+    private $questions;
+
+    public function __construct(Locale $locale, Meta $meta, string $title, int $sequence)
+    {
+        $this->locale = $locale;
+        $this->meta = $meta;
+        $this->title = $title;
+        $this->sequence = $sequence;
+        $this->questions = new ArrayCollection();
```

Comment:
Is this needed?

Reply:
otherwise $this->getQuestions() can be null, what we don't want

Reply:
ok